library/tasks/GroupMetricsOutputs.wdl

```diff
@@ -0,0 +1,52 @@
+task GroupQCOutputs {
+  Array[File] picard_row_outputs
+  Array[File] picard_table_outputs
+  File hisat2_stats
+  File hisat2_trans_stats
+  File rsem_stats
+  String output_name
+  # Runtime
+  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:jx-ss2-group-qc"
+  String mem = 2
+  String cpu = 1
+  String disks = 20 
+  Int preemptible = 5
+  Int max_retries = 0
+  
+  meta {
+    description: "This task will group the Picard metrics"
+  }
+  parameter_meta {
+    picard_outputs: "array of files generated by Picard"
+    hisat2_stats: "statistics output of hisat2 alignment"
+    rsem_stats: "statistics output of rsem "
+    output_name: "name output files"
+    docker: "(optional) the docker image containing the runtime environment for this task"
+    machine_mem_mb: "(optional) the amount of memory (MB) to provision for this task"
+    cpu: "(optional) the number of cpus to provision for this task"
+    disk: "(optional) the amount of disk space (GB) to provision for this task"
```

Comment:
If this is just referring to the disk space and not the `disks` runtime parameter, renaming this to `disk_space` would make that clearer